ext/native-decls/MumbleRemoveVoiceTargetChannel.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: CFX
+apiset: client
+---
+## MUMBLE_REMOVE_VOICE_TARGET_CHANNEL
+
+```c
+void MUMBLE_REMOVE_VOICE_TARGET_CHANNEL(int targetId, int channel);
+```
+
+Removes the voice channel from the users voice targets.
```

Comment:
`the voice channel` -> `a` or `the specified` voice channel
`users` -> `user's`